src/components/ContactUs/submitConfirm.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { ContactCloseBtn } from "./contactUsStyle";
+
+const submitCofirm = props => {
+  return (
+    <div>
+      <h1>Thank you!</h1>
+      <p>
+        Your message has been successfully sent. We will contact you very soon!
+      </p>
```

Comment:
maybe we should put the email here - and say something like if this is urgent or you need to contact us for additional information please contact `admin@codepurpose.io` in a mailto anchor tag

Reply:
That's a good idea. I will add that.